x/bank/spec/README.md

```diff
@@ -22,63 +35,67 @@ supply of all assets used in the application.
 
 This module will be used in the Cosmos Hub.
 
-## Supply
+## 02. Concepts
+
+### Supply
 
 The `supply` module:
```

Comment:
@amaurymartiny Should this refer to the "bank" instaed of "supply" module, as `supply` is not a module anymore? Or maybe referring to as `supply` functionality?

Reply:
Makes sense. These docs were added at the same time as merging `x/supply` into `x/bank` (https://github.com/cosmos/cosmos-sdk/pull/6010), so I thought they made sense. I'll replace with "supply functionality".